scripts/west_commands/runners/jlink.py

```diff
@@ -94,6 +101,8 @@ def do_run(self, command, **kwargs):
                        '-device', self.device,
                        '-silent',
                        '-singlerun'])
+        if self.jlinkscriptfile != '':
```

Comment:
Why is this only needed for debugging, but not flashing?

Reply:
From what I can tell, the defaults for making the RAM accessible are not established when running from the debugger because they are set via the bootloader code. So things crash when the startup code attempts to zero out the bss. So to work around this, the j-link debugger supports a scripting ability to script some pre-initialization stuff in the target before code is run. The script, in this case, came directly from our SDK support.

Reply:
Can it hurt to have this applied for both? Inconsistencies like this make me nervous is all.